uptane-standard.md

```diff
@@ -829,6 +829,12 @@ In order to perform partial verification, an ECU SHALL perform the following ste
 1. Load and verify the current time or the most recent securely attested time.
 2. Download and check the Targets metadata file from the Director repository, following the procedure in {{check_targets}}.
 
+Note that this verification procedure is intended to be a minimum. A partial verification ECU MAY implement more metadata checks.
+
+For example, in many cases it is desirable to also fetch and verify Root metadata from the Director (following the procedure in {{check_root}}) before checking Targets metadata. Without this check, an ECU will not be able to verify any metadata from the Director if the Director's Targets key is rotated, possible necessitating a recall.
```

Comment:
Ok, I see my question has been answered here. Needs to be linked to my question above, IMO.

Reply:
```suggestion
Note that this verification procedure outline the minimal number of actions that must be taken. A partial verification ECU MAY implement more metadata checks.
```

Reply:
I can't insert the suggestion in the right place for some reason. But, I would re-word line 832 as:

Note that this verification procedure outline the minimum number of actions that MUST be performed.